internal/framework/service/list/resource.go

```diff
@@ -99,15 +99,17 @@ func (r *ListResource) Read(ctx context.Context, req resource.ReadRequest, resp
 	state.Description = types.StringValue(list.Description)
 	state.Kind = types.StringValue(list.Kind)
 
-	items, err := r.client.V1.ListListItems(ctx, cloudflare.AccountIdentifier(state.AccountID.ValueString()), cloudflare.ListListItemsParams{
-		ID: state.ID.ValueString(),
-	})
-	if err != nil {
-		resp.Diagnostics.AddError("Error reading List Items", err.Error())
-		return
-	}
+	if len(state.Items) > 0 {
```

Comment:
i don't think this is the equivalent of the [sdkv2 version](https://github.com/cloudflare/terraform-provider-cloudflare/blob/d0edea1ef8e34bec96b84f54632a67fe9e5999c4/internal/sdkv2provider/resource_cloudflare_list.go#L106-L108). if someone starts with zero list items and then adds one, it won't ever get into this block to get the listing.